client/src/App.tsx

```diff
@@ -15,51 +15,58 @@ function App() {
   const [testList, setTestList] = useState<TestResultSet[]>([]);
   const [busy, setBusy] = useState<boolean>(false);
   const [domain, setDomain] = useState("testanchor.stellar.org");
+  const [sepSelect, setSepSelect] = useState<string>("SEP24");
+  const [project, setProject] = useState<string>("SEP24");
   const [currency, setCurrency] = useState<string>("");
   const [runOptionalTests, setRunOptionalTests] = useState<boolean>(
     Boolean(parseInt(process.env.RUN_OPTIONAL_TESTS || "0")) || false,
   );
 
+  const fetchList = async (project: string) => {
+    const res = await fetch(
+      `${process.env.REACT_APP_API_HOST || ""}/list?project=${project}`,
+    );
+    const testNames: string[] = await res.json();
+    setAvailableTests(testNames);
+  };
+
   useEffect(() => {
     const queryString = window.location.search;
     const urlParams = new URLSearchParams(queryString);
     const domain = urlParams.get("home_domain");
     const currency = urlParams.get("currency");
+    const project = urlParams.get("project") || "SEP24";
```

Comment:
could be confusing to shadow the wider `project` var, maybe name this `projectFromUrl` or something